Add RemoveBackground to helpers

jibinmathew69 opened the pull request:
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [x] you only add one (!) new helper per pull request.
- [x] you have checked if an open PR already exists.
- [x] the submitted website is focused on a single, development related issue.
- [x] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").


Thank you! 🙇‍♂️

stefanjudis commented:
Hey @jibinmathew69 thank you for contributing to TH.

I just tried the background remove and it worked only 50% of the time. Every second time it failed to recognize the people and added grey'ish / transparent parts.

<img width="689" alt="image" src="https://github.com/stefanjudis/tiny-helpers/assets/962099/1bbc797f-df99-4ce4-b643-e989688f3cb0">

Could we still improve the outpoint of the background remover?

jibinmathew69 commented:
Thanks Stefanjudis for trying it out and pointing out the issues, yes a
better removal could be done, but would require to run it on a server and
that’s the reason I added an option to “Get API”.



On Sat, 6 Jul 2024 at 9:40 PM, Stefan Judis ***@***.***>
wrote:

> Hey @jibinmathew69 <https://github.com/jibinmathew69> thank you for
> contributing to TH.
>
> I just tried the background remove and it worked only 50% of the time.
> Every second time it failed to recognize the people and added grey'ish /
> transparent parts.
> image.png (view on web)
> <https://github.com/stefanjudis/tiny-helpers/assets/962099/1bbc797f-df99-4ce4-b643-e989688f3cb0>
>
> Could we still improve the outpoint of the background remover?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/stefanjudis/tiny-helpers/pull/539#issuecomment-2211808415>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIGRTHTJW2NGESOERQE7SMLZLAJF3AVCNFSM6AAAAABKOIVZ7GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEMJRHAYDQNBRGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

stefanjudis commented:
@jibinmathew69 Thanks for the explanation. :) 

I'm a bit on the fence in this case. If a helper only works 50% (that's my testing — maybe it's a diff ratio 🫣) I don't think it's a good fit for TH. Another way to succeed (via API as you mentioned) is irrelevant because the idea of TH is to provide free and quick to use tools. 

I hope this makes sense and I'll close here. Feel free to ping me again if the online version has a better "success rate". :)
